src/context-providers/builder/provider.tsx

```diff
@@ -17,6 +17,7 @@ const DEFAULT_VALUES: IBuilderState = {
     focusedElement: null,
     showSidePanel: true,
     orderedIdentifiers: [],
+    isSubmitting: false,
```

Comment:
can we track it within `SidePanel` as a local state instead? because I see it being used in `SidePanelHeader` only